Sources/gg.datagram.web-requests.sdPlugin/app.js

```diff
@@ -18,8 +18,34 @@ function connected(jsn) {
  * }} data
  */
 function sendHttp(data) {
-    const { url, method, body } = data.payload.settings;
-    log('sendHttp', { url, method, body });
+    const { url, method, contentType, headers, body } = data.payload.settings;
+    log('sendHttp', { url, method, contentType, headers, body });
+
+    let defaultHeaders = {
+      'Content-Type': contentType
+    };
+    let inputHeaders = {};
+
+    if (typeof headers !== 'undefined') {
```

Comment:
I'd prefer `if (headers)`. `null` will become a possible value once I fix I bug I just identified while testing this, and there's no point in trying to parse an empty string.